controllers/contactController.js

```diff
@@ -0,0 +1,25 @@
+import Contact from "../models/contact";
+
+class ContactController{
+    static async createContact(req,res){
+        try {
+            const contact = new Contact(req.body);
+            await contact.save();
+            res.status(201).json(contact);
```

Comment:
use the same json form in responses. be consistent
{status, message, data}

Reply:
let me change the form